### PR TITLE
Kp 684

### PR DESCRIPTION
**Pull request by muke1908**
Ticket: https://kycpass.atlassian.net/browse/KP-684

This will accept SDK config from client and pass it to the iframe. 

> `config` will be re-used for further SDK customisation in future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +73,27 @@ Here's a sample of the object that will passed to your `onSuccess` callback:
   }
 }
 ```
+
+### Customisation  
+
+The `config` accepts `prefill` object to pre-fill the SDK form. Only following fields can be prefilled: 
+Legal entity name, Registration code, Country, Representative first name & Representative last name.
+**Note:** `residenceState` is applicable only when `residenceCountry` is `US`.
```
Comment (Merraclius) [inline README.md]: I think we also should add note about format (alpha2) for country and state

**Review by Merraclius — APPROVED** [review]
Only one comment related to README. Other things looks OK

File: README.md [hunk 2]
```diff
@@ -72,6 +73,27 @@ Here's a sample of the object that will passed to your `onSuccess` callback:
   }
 }
 ```
+
+### Customisation  
+
+The `config` accepts `prefill` object to pre-fill the SDK form. Only following fields can be prefilled:
```
Comment (vlasn) [inline README.md]: accepts _a_ prefill object
Only _the_ following fields can be prefilled

File: README.md [hunk 3]
```diff
@@ -72,6 +73,27 @@ Here's a sample of the object that will passed to your `onSuccess` callback:
   }
 }
 ```
+
+### Customisation  
+
+The `config` accepts `prefill` object to pre-fill the SDK form. Only following fields can be prefilled: 
+Legal entity name, Registration code, Country, Representative first name & Representative last name.  
+**Note:** The `residenceState` and `residenceCountry` code has to be **alpha2** format. `residenceState` is applicable only when `residenceCountry` is `US`.
```
Comment (vlasn) [inline README.md]: "have to be in alpha2 format"
"residenceState is only required when residenceCountry is US"

**Pull request merged**